src/pages/admin/TemplateEditorPage.tsx

```
@@ -0,0 +1,453 @@
+  discussionConfig?: DiscussionConfig
+}
+
+interface TemplateDoc {
+  name: string
+  description: string
+  type: TemplateType
+  isGlobal: boolean
+  sections: TemplateSection[]
+}
+
+interface TemplateEditorPageProps {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)